### ISSUE TITLE
Cursor Layer - space and enter?

### ISSUE THREAD

**Issue by AsafMah** [sunaku/glove80-keymaps#6]
The cursor layer helps to use one half plus mouse, but the lack of enter and space on it in general, and specifically on the left hand feels like an oversight

also I'm unsure on how to use the thumb alt tab and such keys, since the thumb is already at the layer button normally, and even when locking, to progress in alt tab you need to use the tab key which is also a thumb key.

**Comment by sunaku** [comment]
Thanks for your feedback.  I've added some Spacegram operators on the left half of the keyboard in [this development snapshot](https://my.glove80.com/#/layout/user/53abaa35-5936-4651-9e01-a7fe8bc648f4).

As for Alt+Tab, I've replaced the explicit layer chord key on the Number layer with a proper ZMK combo so that it doesn't matter which key you press in the chord first: you'll be momentarily taken to the cursor layer while the Alt+Tab combo is being held down.  Once you're there, the arrow keys on the right hand's home row can help navigate between windows in the Alt+Tab menu -- you don't need the tab key, per se, for such navigation (at least in Linux and Windows according to my observations).
